modules/apps/commerce/commerce-product-test/src/testFunctional/macros/FrontStore.macro

```diff
@@ -545,7 +545,7 @@ definition {
 	}
 
 	@summary = "Default summary"
-	macro gotoEditMiniCartItemBundledProduct(cartItemSku = null) {
+	macro gotoEditMiniCartItemOptionProduct(cartItemSku = null) {
```

Comment:
The product option is being targeted, right? We may name the macro like this:
gotoEditMiniCartItemProductOption